src/utils/permalinks/Permalinks.ts

```diff
@@ -74,10 +74,19 @@ const MAX_SERVER_CANDIDATES = 3;
 // the list and magically have the link work.
 
 export class RoomPermalinkCreator {
+    private _room: Room;
+    private _roomId: string;
+    private _highestPlUserId: string;
+    private _populationMap: { [serverName: string]: number };
+    private _bannedHostsRegexps: RegExp[];
+    private _allowedHostsRegexps: RegExp[];
+    private _serverCandidates: string[];
+    private _started: boolean;
```

Comment:
any reason these remain prefixed? we seem to tend to avoid that in TS where we have an explicit `private` keyword

Reply:
Have fixed all except `_serverCandidates` since there's a getter with same name.